tcms/testruns/models.py

```diff
@@ -26,7 +26,7 @@
 class TestRun(models.Model, UrlMixin):
     history = KiwiHistoricalRecords()
 
-    start_date = models.DateTimeField(auto_now_add=True, db_index=True)
+    start_date = models.DateTimeField(db_index=True, null=True, blank=True)
```

Comment:
See `sanity / test-for-missing-migrations` .  I believe this is what is causing the rest of the tests to fail as well.